### DISCUSSION TITLE
java.nio.file.AccessDeniedException in JetBrains

### DISCUSSION THREAD

**Discussion by pasan1** [community/community#57040] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Hi,

When I am going to uninstall the Copilot this error occured

The cause: java.nio.file.AccessDeniedException: C:\Users\pasan\AppData\Local\JetBrains\Toolbox\apps\IDEA-U\ch-0\223.8836.41.plugins\github-copilot-intellij\copilot-agent\bin\copilot-agent-win.exe

Thank you

**Comment by bcb44** [comment]
This is still happening - why do we even need a native library for this??? The llm is all openai apis anyway
<img width="987" alt="Screenshot 2024-06-12 at 6 58 36 AM" src="https://github.com/community/community/assets/42728060/59ba7a2c-26c4-4343-82a8-4c5e735da542">
<img width="384" alt="Screenshot 2024-06-12 at 6 59 31 AM" src="https://github.com/community/community/assets/42728060/223544e3-be2c-4b69-8fcf-b35089c59d34">

**Comment by ilysharusher** [comment]
I've been having the same problem for six months now with PhpStorm. It seems that the developers forgot that they released Copilot not only for VsCode. They also demand money for something that is not updated
